### DISCUSSION TITLE
レイヤーを移動した際のLED照明色の変更について

### DISCUSSION THREAD

**Discussion by apio1969** [Yowkees/keyball#233] [General]
お世話になります。
keyball61を使わせて頂いております。LAND剥がれ補修後は不具合無く動作しています。
ありがとうございます。
今日はLEDの点灯について伺いたく書き込みしました。
keyball61使用中にて、レイヤーを移動した際にLEDの点灯色を変更することは可能ですか。
レイヤー2に数字と矢印キーなどを割り当てています。これをTO(2)にてレイヤーを移動させているのですが、
レイヤー別にLEDを色分けすることができれば、視覚的にどのレイヤーにいるのかわかるかと思います。
色分けが不可能であれば、LEDを消灯するとかは可能でしょうか。
もし、色分け、もしくは消灯が可能であれば、その方法をご教授頂けると幸いです。
おいそがしい事と思いますが、お時間のあるときでけっこうです。ご教授頂けると幸いです。
よろしくお願いいたします。m(_ _)m

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
レイヤー切り替え時のLED色を切り替えるファーム改造は、私はまだトライ出来ていないのですが、
何人かの方が挑戦されていて、ファームウェアを公開されている方もみえます。
例えば下記の記事が参考になりませんでしょうか。
https://note.com/mickie895/n/n2bb4e20fd48c
ご検討お願いいたしますm(_ _)m

**Comment by apio1969** [comment]
ヨーキーズさん、お忙しい中ご返答ありがとうございます。
内容については了解しました。
送っていただいたURL等を参考にして見たいと思います。
敷居が高そうですが、できそうならチャレンジして妙かと思います。
ありがとうございました。
